src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -312,9 +328,9 @@ func statusShouldBeUpdated(
 
 // WithConfiguratorTag set the configuratorTag
 func (r *ClusterReconciler) WithConfiguratorTag(
```

Comment:
Should we change this to `WithConfiguratorImage` or `WithConfiguratorSettings`?

Reply:
Thank you, yes :)